goss/flatcar/command.yaml

```diff
@@ -0,0 +1,11 @@
+command:
+  version:
+    exit-status: 0
+    exec: "kubeadm version -o short"
+    # update this when we change kubernetes version to `
+    stdout:
+      - v1.25.4
```

Comment:
The kubernetes version is not strictly fixed in all usecases. We allow to override using CLI flag and override files too. 
is this any reason this is strictly needed for flatcar?